src/main/java/com/powsybl/ws/commons/SecuredInputStream.java

```diff
@@ -11,14 +11,14 @@
  */
 public class SecuredInputStream {
     private final int maxEntries;
```

Comment:
```suggestion
    // This attribute defines the maximum of entries count an archive can contain. This limit is defined to avoid zip/tar bombs.
    private final int maxEntries;
```